client/state/partner-portal/licenses/hooks/use-products-query.ts

```diff
@@ -49,11 +50,14 @@ function queryProducts(): Promise< APIProductFamily[] > {
 export default function useProductsQuery(): UseQueryResult< APIProductFamilyProduct[], unknown > {
 	const translate = useTranslate();
 	const dispatch = useDispatch();
+	const isPartnerOAuthTokenLoaded = useSelector( getIsPartnerOAuthTokenLoaded );
 
 	const query = useQuery( {
 		queryKey: [ 'partner-portal', 'licenses', 'products' ],
 		queryFn: queryProducts,
 		select: selectAlphabeticallySortedProductOptions,
+		enabled: isPartnerOAuthTokenLoaded,
```

Comment:
I wonder why we didn't add this before since the problem you're solving was probably present in other places where we use the `useProductsQuery` hook.

Said differently, could we be breaking something else by putting this condition on the hook? Is there a place where that condition is not needed?

Reply:
Yashwin pointed me out with that solution.

Reply:
Disregard my comment, then.